doc/changes/DM-43381.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed bug preventing ``bps report`` from showing error codes/counts correctly when called with ``--return-exit-codes``.
```

Comment:
The description doesn't match the ticket where it fails when using ``--id <path>``